v1.1/known-limitations.md

```diff
@@ -1,46 +1,94 @@
 - A table cannot be dropped and then recreated with the same name. This is not possible within a single transaction because `DROP TABLE` does not immediately drop the name of the table. As a workaround, split the [`DROP TABLE`](drop-table.html) and [`CREATE TABLE`](create-table.html) statements into separate transactions.
 
-## Join flags when restoring a backup onto new machines
+### Schema changes between executions of prepared statements
+
+When the schema of a table targeted by a prepared statement changes before the prepared statement is executed, CockroachDB should return an error. Instead, CockroachDB incorrectly allows the prepared statement to return results based on the changed table schema, for example:
+
+~~~ sql
+> CREATE TABLE users (id INT PRIMARY KEY);
+
+> PREPARE prep1 AS SELECT * FROM users;
+
+> ALTER TABLE users ADD COLUMN name STRING;
+
+> INSERT INTO users VALUES (1, 'Max Roach');
 
-In our [deployment tutorials](manual-deployment.html), when starting the first node of a cluster, the `--join` flag should be empty, but when starting all subsequent nodes, the `--join` flag should be set to the address of node 1. This approach ensures that all nodes have access to a copy of the first key-value range, which is part of a meta-index identifying where all range replicas are stored, and which nodes require to initialize themselves and start accepting incoming connections.
+> EXECUTE prep1;
+~~~
+
+~~~
++----+-----------+
+| id |   name    |
++----+-----------+
+|  1 | Max Roach |
++----+-----------+
+(1 row)
+~~~
+
+Also, a prepared [`INSERT`](insert.html), [`UPSERT`](upsert.html), or [`DELETE`](delete.html) statement acts inconsistently when the schema of the table being written to is changed before the prepared statement is executed:
+
+- If the number of columns has increased, the prepared statement returns an error but nonetheless writes the data.
+- If the number of columns remains the same but the types have changed, the prepared statement writes the data and does not return an error.
+
+### Join flags when restoring a backup onto new machines
+
+All nodes in a cluster must have access to a copy of the first key-value range, which is part of a meta-index identifying where all range replicas are stored, and which nodes require to initialize themselves and start accepting incoming connections.
 
 Ensuring that all nodes have access to a copy of the first key-value range is more difficult when restoring from a whole-cluster backup onto machines with different IP addresses than the original cluster. In this case, the `--join` flags must form a fully-connected directed graph. The easiest way to do this is to put all of the new nodes' addresses into each node's `--join` flag, which ensures all nodes can join a node with a copy of the first key-value range.
```

Comment:
I'd recommend putting the first 3-5 nodes as join targets, not all the nodes. 

This is recommended in all cases, not just when restoring from a backup (we're even talking about making it mandatory in the future).

Reply:
Since this case is now not different than any other, I'd say this isn't known limitation anymore. I'm removing, but I can put it back post-release, if you can help me understand how to phrase it so it's a limitation.